### PR TITLE
make group for subjects

### PR DESCRIPTION
**Pull request by thiennv57**

### CODE REVIEW

File: app/controllers/admin/courses_controller.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def create
   def update
     if @course.update_attributes course_params
       flash[:success] = flash_message "updated"
```
Comment (yongsokheng) [inline app/controllers/admin/courses_controller.rb]: I18n

File: app/models/group_subject.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class GroupSubject < ActiveRecord::Base
   has_many :user_subjects
   has_many :users, through: :user_subjects
+
+  validates :name, presence: true
```
Comment (yongsokheng) [inline app/models/group_subject.rb]: them validate unique nua nha cau

**Comment by NgocVu1994** [comment]
OK

**Comment by yongsokheng** [comment]
OK

**Pull request merged**